### ISSUE TITLE
Add Sort and Filter feature

### ISSUE THREAD

**Issue by rishavjaiss** [rishavjaiss/movie-search#4]
Hi,

This issue is opened to implement sorting and filter feature to the results obtained on search.
*Code of Conduct* To be followed strictly : You should generate your own API key from here - https://www.themoviedb.org/documentation/api and only use that.

Happy Hacktober!

**Comment by cjharkins** [comment]
I'll take it!

**Comment by rishavjaiss** [comment]
Thanks for your interest. Please follow the code of conduct @cjharkins. Assigning this to you.

**Comment by cjharkins** [comment]
@rishavjaiss you are aware that your API key is exposed?

**Comment by rishavjaiss** [comment]
Oh no! This project was untouched for long. Thanks for pointing out. I have removed it @cjharkins. Start working on it.

**Comment by cjharkins** [comment]
@rishavjaiss can you check the permissions on this repository. I'm trying to push up a new branch for PR and I'm getting a 403 (unauthorized). Thanks!

**Comment by rishavjaiss** [comment]
@cjharkins make the PR to the main branch. This shouldn't give an error :/

**Comment by cjharkins** [comment]
@rishavjaiss I am trying to push my local branch up to this repo so I can make a PR

**Comment by rishavjaiss** [comment]
Haven't you forked the repo before making changes? I guess if you fork and make changes and then make the pr that shouldn't give any error. I am new to open source collaboration. Please check.

**Comment by cjharkins** [comment]
I did not fork, would you like me to fork it or contribute directly to this project?

**Comment by rishavjaiss** [comment]
Ah yes. I don't know if there's other better way to do so? I have myself been doing like this so i know this one way only.

**Comment by rishavjaiss** [comment]
Hey @cjharkins , really appreciate your work. I read on the internet that it's better to make a branch and keep your version of code separate. I can see you are using xml. Please give me some time I'll check how to allow contributors to make branches and make pr to that and then I'll notify you. Your involvement helped me learn something new on GitHub! Thanks man.

**Comment by cjharkins** [comment]
Hey! No worries. 

Any XML outside of ReactJSX shouldn't be in there, you may be viewing older commits, ensure you are looking at the most recent commits and updates for that branch. There were some .idea files that made it's way into my commit I have deleted those files as well as added to .gitignore.

**Comment by rishavjaiss** [comment]
Yes I have now seen the latest code which you made the pr with. I have completely understood the approach and the code. But I was thinking to try the branching way which I believe is most feasible and professional as that will keep my version intact and your features separate. I'll notify you asap then can you make the pr to a separate branch? Thanks for your contribution.
